install.sh

```diff
@@ -4,11 +4,20 @@ set -e
 set -o noglob
 
 ####################################################################### funcs
+test_exists() {
+  if which which >/dev/null 2>&1; then
+    which "$1" >/dev/null 2>&1
+  else
+    #FIXME not everything has `--version`
+    $1 --version >/dev/null 2>&1
+  fi
```

Comment:
```suggestion
  command -v $1 >/dev/null
```

Reply:
good find. confirmed working in archlinux.